app/admin/quests/create/page.tsx

```diff
@@ -383,6 +383,19 @@ export default function Page() {
         } catch (error) {
           console.error("Error while creating balance task:", error);
         }
+      } else if (step.type === "Contract"){
+        try {
+          await AdminService.createContract({
+            quest_id: questId,
+            name: step.data.contract_name,
+            desc: step.data.contract_desc,
+            href: step.data.contract_href,
+            cta: step.data.contract_cta,
+            calls: step.data.contract_calls,
+          });
+        } catch (error) {
+          console.error("Error while creating contract task:", error);
```

Comment:
Missing 
```tsx
showNotification(`Error adding ${step.type} task: ${error}`, "error");
``` here (should not only throw an error in the console, but also tell that there is an issue directly on the screen using our notification system).